actor/actionerror/service_plan_not_found_error.go

```diff
@@ -0,0 +1,12 @@
+package actionerror
+
+import "fmt"
+
+type ServicePlanNotFoundError struct {
+	PlanName        string
+	ServiceName     string
+}
+
+func (e ServicePlanNotFoundError) Error() string {
+	return fmt.Sprintf("The plan '%s' could not be found for service %s", e.PlanName, e.ServiceName)
```

Comment:
Is the inconsistent quote between the plan name and service name to match the old code? or can this be fixed?

Reply:
I think it's a mistake. The quotes are not present neither in the old code, nor in the translated error. I'll remove them